### PR TITLE
feat: add ping connectivity

### LINKED ISSUE #36: Add support for additional connectivity health checks
### Code of Conduct

- [X] I have read and agree to the project's Code of Conduct.
- [X] Vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue initial description to help the maintainers prioritize.
- [X] Do not leave "+1" or other comments that do not add relevant information or questions.
- [X] If you are interested in working on this issue or have submitted a pull request, please leave a comment.

### Description

Add support for including the results of `"Connectivity": { "Ping Status": {` in the Connectivity Health section of the Health Report.

### Use Case(s)

Provides the status of ICMP requests to the IP of:

- Platform Components (vCenter Server, ESXi Hosts, NSX Managers (Local)
- vRealize Suite Components, if in VCF-mode.

### Potential Configuration

To be determined.

### References

Example JSON

```json
    "Connectivity": {
        "Ping Status": {
            "sfo01-w01-esx01.sfo.rainpole.io": {
                "172.18.103.101": {
                    "area": "ESXi : sfo01-w01-esx01.sfo.rainpole.io",
                    "title": "Ping status",
                    "state": "REACHABLE",
                    "timestamp": "Mon Dec  5 23:24:10 2022",
                    "message": "Ping successful!",
                    "status": "passed",
                    "alert": "GREEN"
                }
            },
            "sfo01-w01-esx03.sfo.rainpole.io": {
                "172.18.103.103": {
                    "area": "ESXi : sfo01-w01-esx03.sfo.rainpole.io",
                    "title": "Ping status",
                    "state": "REACHABLE",
                    "timestamp": "Mon Dec  5 23:24:10 2022",
                    "message": "Ping successful!",
                    "status": "passed",
                    "alert": "GREEN"
                }
            },
            "sfo01-w01-esx04.sfo.rainpole.io": {
                "172.18.103.104": {
                    "area": "ESXi : sfo01-w01-esx04.sfo.rainpole.io",
                    "title": "Ping status",
                    "state": "REACHABLE",
                    "timestamp": "Mon Dec  5 23:24:10 2022",
                    "message": "Ping successful!",
                    "status": "passed",
                    "alert": "GREEN"
                }
            },
            "sfo-w01-vc01.sfo.rainpole.io": {
                "area": "vCenter : sfo-w01-vc01.sfo.rainpole.io",
                "title": "Ping status",
                "state": "REACHABLE",
                "timestamp": "Mon Dec  5 23:24:13 2022",
                "message": "Ping successful!",
                "status": "passed",
                "alert": "GREEN"
            },
            "172.18.95.131": {
                "area": "NSX: sfo-w01-nsx01.sfo.rainpole.io",
                "title": "Ping status",
                "state": "REACHABLE",
                "timestamp": "Mon Dec  5 23:24:13 2022",
                "message": "Ping successful!",
                "status": "passed",
                "alert": "GREEN"
            },
            "xint-vra01a.rainpole.io": {
                "area": "VRA : xint-vra01a.rainpole.io",
                "title": "Ping status",
                "state": "REACHABLE",
                "timestamp": "Mon Dec  5 23:24:13 2022",
                "message": "Ping successful!",
                "status": "passed",
                "alert": "GREEN"
            },
            "xint-vra01b.rainpole.io": {
                "area": "VRA : xint-vra01b.rainpole.io",
                "title": "Ping status",
                "state": "REACHABLE",
                "timestamp": "Mon Dec  5 23:24:13 2022",
                "message": "Ping successful!",
                "status": "passed",
                "alert": "GREEN"
            },
            "xint-vra01c.rainpole.io": {
                "area": "VRA : xint-vra01c.rainpole.io",
                "title": "Ping status",
                "state": "REACHABLE",
                "timestamp": "Mon Dec  5 23:24:13 2022",
                "message": "Ping successful!",
                "status": "passed",
                "alert": "GREEN"
            },
            "xint-vrslcm01.rainpole.io": {
                "area": "VRSLCM : xint-vrslcm01.rainpole.io",
                "title": "Ping status",
                "state": "REACHABLE",
                "timestamp": "Mon Dec  5 23:24:13 2022",
                "message": "Ping successful!",
                "status": "passed",
                "alert": "GREEN"
            },
            "sfo-vrli01a.sfo.rainpole.io": {
                "area": "vRLI : sfo-vrli01a.sfo.rainpole.io",
                "title": "Ping status",
                "state": "REACHABLE",
                "timestamp": "Mon Dec  5 23:24:13 2022",
                "message": "Ping successful!",
                "status": "passed",
                "alert": "GREEN"
            },
            "sfo-vrli01b.sfo.rainpole.io": {
                "area": "vRLI : sfo-vrli01b.sfo.rainpole.io",
                "title": "Ping status",
                "state": "REACHABLE",
                "timestamp": "Mon Dec  5 23:24:13 2022",
                "message": "Ping successful!",
                "status": "passed",
                "alert": "GREEN"
            },
            "sfo-vrli01c.sfo.rainpole.io": {
                "area": "vRLI : sfo-vrli01c.sfo.rainpole.io",
                "title": "Ping status",
                "state": "REACHABLE",
                "timestamp": "Mon Dec  5 23:24:13 2022",
                "message": "Ping successful!",
                "status": "passed",
                "alert": "GREEN"
            },
            "xint-vrops01a.rainpole.io": {
                "area": "vROps : xint-vrops01a.rainpole.io",
                "title": "Ping status",
                "state": "REACHABLE",
                "timestamp": "Mon Dec  5 23:24:13 2022",
                "message": "Ping successful!",
                "status": "passed",
                "alert": "GREEN"
            },
            "xint-vrops01b.rainpole.io": {
                "area": "vROps : xint-vrops01b.rainpole.io",
                "title": "Ping status",
                "state": "REACHABLE",
                "timestamp": "Mon Dec  5 23:24:13 2022",
                "message": "Ping successful!",
                "status": "passed",
                "alert": "GREEN"
            },
            "sfo-vropsc01a.sfo.rainpole.io": {
                "area": "vROps : sfo-vropsc01a.sfo.rainpole.io",
                "title": "Ping status",
                "state": "REACHABLE",
                "timestamp": "Mon Dec  5 23:24:13 2022",
                "message": "Ping successful!",
                "status": "passed",
                "alert": "GREEN"
            },
            "sfo-vropsc01b.sfo.rainpole.io": {
                "area": "vROps : sfo-vropsc01b.sfo.rainpole.io",
                "title": "Ping status",
                "state": "REACHABLE",
                "timestamp": "Mon Dec  5 23:24:13 2022",
                "message": "Ping successful!",
                "status": "passed",
                "alert": "GREEN"
            },
            "xint-vrops01c.rainpole.io": {
                "area": "vROps : xint-vrops01c.rainpole.io",
                "title": "Ping status",
                "state": "REACHABLE",
                "timestamp": "Mon Dec  5 23:24:13 2022",
                "message": "Ping successful!",
                "status": "passed",
                "alert": "GREEN"
            },
            "xint-wsa01a.rainpole.io": {
                "area": "WSA : xint-wsa01a.rainpole.io",
                "title": "Ping status",
                "state": "REACHABLE",
                "timestamp": "Mon Dec  5 23:24:16 2022",
                "message": "Ping successful!",
                "status": "passed",
                "alert": "GREEN"
            },
            "xint-wsa01b.rainpole.io": {
                "area": "WSA : xint-wsa01b.rainpole.io",
                "title": "Ping status",
                "state": "REACHABLE",
                "timestamp": "Mon Dec  5 23:24:16 2022",
                "message": "Ping successful!",
                "status": "passed",
                "alert": "GREEN"
            },
            "xint-wsa01c.rainpole.io": {
                "area": "WSA : xint-wsa01c.rainpole.io",
                "title": "Ping status",
                "state": "REACHABLE",
                "timestamp": "Mon Dec  5 23:24:16 2022",
                "message": "Ping successful!",
                "status": "passed",
                "alert": "GREEN"
            }
        },
        "Connectivity Status": {
            "ESXi API Status": {
                "sfo01-w01-esx01.sfo.rainpole.io": {
                    "area": "ESXi : sfo01-w01-esx01.sfo.rainpole.io",
                    "title": "API Connectivity status",
                    "state": "REACHABLE",
                    "timestamp": "Mon Dec  5 23:24:10 2022",
                    "message": "API Connectivity check successful!",
                    "status": "passed",
                    "alert": "GREEN"
                },
                "sfo01-w01-esx03.sfo.rainpole.io": {
                    "area": "ESXi : sfo01-w01-esx03.sfo.rainpole.io",
                    "title": "API Connectivity status",
                    "state": "REACHABLE",
                    "timestamp": "Mon Dec  5 23:24:10 2022",
                    "message": "API Connectivity check successful!",
                    "status": "passed",
                    "alert": "GREEN"
                },
                "sfo01-w01-esx04.sfo.rainpole.io": {
                    "area": "ESXi : sfo01-w01-esx04.sfo.rainpole.io",
                    "title": "API Connectivity status",
                    "state": "REACHABLE",
                    "timestamp": "Mon Dec  5 23:24:10 2022",
                    "message": "API Connectivity check successful!",
                    "status": "passed",
                    "alert": "GREEN"
                }
            },
            "ESXi SSH Status": {
                "sfo01-w01-esx01.sfo.rainpole.io": {
                    "area": "ESXi : sfo01-w01-esx01.sfo.rainpole.io",
                    "title": "** SSH status is disabled.",
                    "state": "NOT_RUNNING",
                    "timestamp": "Mon Dec  5 23:24:10 2022",
                    "message": "** SSH status is disabled.",
                    "status": "passed",
                    "alert": "-"
                },
                "sfo01-w01-esx03.sfo.rainpole.io": {
                    "area": "ESXi : sfo01-w01-esx03.sfo.rainpole.io",
                    "title": "** SSH status is disabled.",
                    "state": "NOT_RUNNING",
                    "timestamp": "Mon Dec  5 23:24:10 2022",
                    "message": "** SSH status is disabled.",
                    "status": "passed",
                    "alert": "-"
                },
                "sfo01-w01-esx04.sfo.rainpole.io": {
                    "area": "ESXi : sfo01-w01-esx04.sfo.rainpole.io",
                    "title": "** SSH status is disabled.",
                    "state": "NOT_RUNNING",
                    "timestamp": "Mon Dec  5 23:24:10 2022",
                    "message": "** SSH status is disabled.",
                    "status": "passed",
                    "alert": "-"
                }
            },
            "sfo-w01-vc01.sfo.rainpole.io": {
                "area": "vCenter : sfo-w01-vc01.sfo.rainpole.io",
                "title": "** SSH status is enabled.",
                "state": "REACHABLE",
                "timestamp": "Mon Dec  5 23:24:10 2022",
                "message": "** SSH status is enabled.",
                "status": "passed",
                "alert": "-"
            },
            "sfo-w01-nsx01.sfo.rainpole.io": {
                "area": "NSX: sfo-w01-nsx01.sfo.rainpole.io",
                "title": "API Connectivity status",
                "state": "REACHABLE",
                "timestamp": "Mon Dec  5 23:24:10 2022",
                "message": "API Connection check successful!",
                "status": "passed",
                "alert": "GREEN"
            },
            "xint-vrslcm01.rainpole.io": {
                "area": "VRSLCM : xint-vrslcm01.rainpole.io",
                "title": "** SSH status is enabled.",
                "state": "REACHABLE",
                "timestamp": "Mon Dec  5 23:24:10 2022",
                "message": "** SSH status is enabled.",
                "status": "passed",
                "alert": "-"
            },
            "sfo-vrli01a.sfo.rainpole.io": {
                "area": "VRLI : sfo-vrli01a.sfo.rainpole.io",
                "title": "** SSH status is enabled.",
                "state": "REACHABLE",
                "timestamp": "Mon Dec  5 23:24:10 2022",
                "message": "** SSH status is enabled.",
                "status": "passed",
                "alert": "-"
            },
            "sfo-vrli01b.sfo.rainpole.io": {
                "area": "VRLI : sfo-vrli01b.sfo.rainpole.io",
                "title": "** SSH status is enabled.",
                "state": "REACHABLE",
                "timestamp": "Mon Dec  5 23:24:10 2022",
                "message": "** SSH status is enabled.",
                "status": "passed",
                "alert": "-"
            },
            "sfo-vrli01c.sfo.rainpole.io": {
                "area": "VRLI : sfo-vrli01c.sfo.rainpole.io",
                "title": "** SSH status is enabled.",
                "state": "REACHABLE",
                "timestamp": "Mon Dec  5 23:24:10 2022",
                "message": "** SSH status is enabled.",
                "status": "passed",
                "alert": "-"
            },
            "xint-vra01a.rainpole.io": {
                "area": "VRA : xint-vra01a.rainpole.io",
                "title": "** SSH status is enabled.",
                "state": "REACHABLE",
                "timestamp": "Mon Dec  5 23:24:10 2022",
                "message": "** SSH status is enabled.",
                "status": "passed",
                "alert": "-"
            },
            "xint-vra01b.rainpole.io": {
                "area": "VRA : xint-vra01b.rainpole.io",
                "title": "** SSH status is enabled.",
                "state": "REACHABLE",
                "timestamp": "Mon Dec  5 23:24:10 2022",
                "message": "** SSH status is enabled.",
                "status": "passed",
                "alert": "-"
            },
            "xint-vra01c.rainpole.io": {
                "area": "VRA : xint-vra01c.rainpole.io",
                "title": "** SSH status is enabled.",
                "state": "REACHABLE",
                "timestamp": "Mon Dec  5 23:24:10 2022",
                "message": "** SSH status is enabled.",
                "status": "passed",
                "alert": "-"
            },
            "xint-vrops01a.rainpole.io": {
                "area": "vROps : xint-vrops01a.rainpole.io",
                "title": "** SSH status is enabled.",
                "state": "REACHABLE",
                "timestamp": "Mon Dec  5 23:24:10 2022",
                "message": "** SSH status is enabled.",
                "status": "passed",
                "alert": "-"
            },
            "xint-vrops01b.rainpole.io": {
                "area": "vROps : xint-vrops01b.rainpole.io",
                "title": "** SSH status is enabled.",
                "state": "REACHABLE",
                "timestamp": "Mon Dec  5 23:24:10 2022",
                "message": "** SSH status is enabled.",
                "status": "passed",
                "alert": "-"
            },
            "sfo-vropsc01a.sfo.rainpole.io": {
                "area": "vROps : sfo-vropsc01a.sfo.rainpole.io",
                "title": "** SSH status is enabled.",
                "state": "REACHABLE",
                "timestamp": "Mon Dec  5 23:24:10 2022",
                "message": "** SSH status is enabled.",
                "status": "passed",
                "alert": "-"
            },
            "sfo-vropsc01b.sfo.rainpole.io": {
                "area": "vROps : sfo-vropsc01b.sfo.rainpole.io",
                "title": "** SSH status is enabled.",
                "state": "REACHABLE",
                "timestamp": "Mon Dec  5 23:24:10 2022",
                "message": "** SSH status is enabled.",
                "status": "passed",
                "alert": "-"
            },
            "xint-vrops01c.rainpole.io": {
                "area": "vROps : xint-vrops01c.rainpole.io",
                "title": "** SSH status is enabled.",
                "state": "REACHABLE",
                "timestamp": "Mon Dec  5 23:24:10 2022",
                "message": "** SSH status is enabled.",
                "status": "passed",
                "alert": "-"
            },
            "xint-wsa01a.rainpole.io": {
                "area": "WSA : xint-wsa01a.rainpole.io",
                "title": "** SSH status is enabled.",
                "state": "REACHABLE",
                "timestamp": "Mon Dec  5 23:24:13 2022",
                "message": "** SSH status is enabled.",
                "status": "passed",
                "alert": "-"
            },
            "xint-wsa01b.rainpole.io": {
                "area": "WSA : xint-wsa01b.rainpole.io",
                "title": "** SSH status is enabled.",
                "state": "REACHABLE",
                "timestamp": "Mon Dec  5 23:24:13 2022",
                "message": "** SSH status is enabled.",
                "status": "passed",
                "alert": "-"
            },
            "xint-wsa01c.rainpole.io": {
                "area": "WSA : xint-wsa01c.rainpole.io",
                "title": "** SSH status is enabled.",
                "state": "REACHABLE",
                "timestamp": "Mon Dec  5 23:24:13 2022",
                "message": "** SSH status is enabled.",
                "status": "passed",
                "alert": "-"
            }
        }
    },
```

### PR DESCRIPTION
**Pull request by tenthirtyam**
In order to have a good experience with our community, we recommend that you read the [contributing guidelines](https://github.com/vmware/powershell-module-for-cloud-foundation-reorting/blob/main/CONTRIBUTING_DCO.md) for making a pull request.

**Summary of Pull Request**

- Added `Publish-PingConnectivityHealth` to return the ping connectivity health from the SoS Health Summary JSON data.
- Updated `Publish-ComponentConnectivityHealth` to include the ping connectivity health using the `Publish-PingConnectivityHealth` cmdlet.
- Updated `CHANGELOG.md`.
- Updated module version from v2.1.0.1003 to v2.1.0.1004.

In order to have a good experience with our community, we recommend that you read the [contributing guidelines]

**Type of Pull Request**

- [ ] This is a bug fix.
- [x] This is an enhancement or feature.
- [ ] This is a code style / formatting update.
- [ ] This is a documentation update.
- [ ] This is a refactoring update.
- [ ] This is a chore update
- [ ] This is something else.
      Please describe:

**Related to Existing Issues**

Closes #36

**Test and Documentation Coverage**

- [x] Tests have been completed (for bug fixes / features).
- [ ] Documentation has been added / updated (for bug fixes / features).

```powershell
PS F:\Reporting\HealthReports> Publish-PingConnectivityHealth -json .\sfo-vcf01-all-health-results.json

Component Resource                        Alert Message
--------- --------                        ----- -------
ESXi      sfo01-m01-esx01.sfo.rainpole.io GREEN Ping successful!
ESXi      sfo01-m01-esx02.sfo.rainpole.io GREEN Ping successful!
ESXi      sfo01-m01-esx03.sfo.rainpole.io GREEN Ping successful!
ESXi      sfo01-m01-esx04.sfo.rainpole.io GREEN Ping successful!
ESXi      sfo01-w01-esx01.sfo.rainpole.io GREEN Ping successful!
ESXi      sfo01-w01-esx02.sfo.rainpole.io GREEN Ping successful!
ESXi      sfo01-w01-esx03.sfo.rainpole.io GREEN Ping successful!
ESXi      sfo01-w01-esx04.sfo.rainpole.io GREEN Ping successful!
NSX       sfo-m01-nsx01.sfo.rainpole.io   GREEN Ping successful!
NSX       sfo-w01-nsx01.sfo.rainpole.io   GREEN Ping successful!
vCenter   sfo-m01-vc01.sfo.rainpole.io    GREEN Ping successful!
vCenter   sfo-w01-vc01.sfo.rainpole.io    GREEN Ping successful!
```

```powershell
PS F:\> Publish-ComponentConnectivityHealth -server sfo-vcf01.sfo.rainpole.io -user admin@local -pass VMw@re1!VMw@re1! -json F:\Reporting\HealthReports\sfo-vcf01-all-health-results.json -allDo
<a id="general-connectivity"></a><h3>Connectivity Health Status</h3>
<table class="table">
<colgroup><col/><col/><col/><col/></colgroup>
<tr><th>Component</th><th>Resource</th><th>Alert</th><th>Message</th></tr>
<tr><td>ESXi</td><td>sfo01-m01-esx01.sfo.rainpole.io</td><td class="alertOK">GREEN</td><td>API Connectivity check successful!</td></tr>
<tr><td>ESXi</td><td>sfo01-m01-esx01.sfo.rainpole.io</td><td>-</td><td>** SSH status is disabled.</td></tr>
<tr><td>ESXi</td><td>sfo01-m01-esx01.sfo.rainpole.io</td><td class="alertOK">GREEN</td><td>Ping successful!</td></tr>
<tr><td>ESXi</td><td>sfo01-m01-esx02.sfo.rainpole.io</td><td class="alertOK">GREEN</td><td>API Connectivity check successful!</td></tr>
<tr><td>ESXi</td><td>sfo01-m01-esx02.sfo.rainpole.io</td><td>-</td><td>** SSH status is disabled.</td></tr>
<tr><td>ESXi</td><td>sfo01-m01-esx02.sfo.rainpole.io</td><td class="alertOK">GREEN</td><td>Ping successful!</td></tr>
<tr><td>ESXi</td><td>sfo01-m01-esx03.sfo.rainpole.io</td><td>-</td><td>** SSH status is disabled.</td></tr>
<tr><td>ESXi</td><td>sfo01-m01-esx03.sfo.rainpole.io</td><td class="alertOK">GREEN</td><td>Ping successful!</td></tr>
<tr><td>ESXi</td><td>sfo01-m01-esx03.sfo.rainpole.io</td><td class="alertOK">GREEN</td><td>API Connectivity check successful!</td></tr>
<tr><td>ESXi</td><td>sfo01-m01-esx04.sfo.rainpole.io</td><td class="alertOK">GREEN</td><td>API Connectivity check successful!</td></tr>
<tr><td>ESXi</td><td>sfo01-m01-esx04.sfo.rainpole.io</td><td class="alertOK">GREEN</td><td>Ping successful!</td></tr>
<tr><td>ESXi</td><td>sfo01-m01-esx04.sfo.rainpole.io</td><td>-</td><td>** SSH status is disabled.</td></tr>
<tr><td>ESXi</td><td>sfo01-w01-esx01.sfo.rainpole.io</td><td>-</td><td>** SSH status is disabled.</td></tr>
<tr><td>ESXi</td><td>sfo01-w01-esx01.sfo.rainpole.io</td><td class="alertOK">GREEN</td><td>Ping successful!</td></tr>
<tr><td>ESXi</td><td>sfo01-w01-esx01.sfo.rainpole.io</td><td class="alertOK">GREEN</td><td>API Connectivity check successful!</td></tr>
<tr><td>ESXi</td><td>sfo01-w01-esx03.sfo.rainpole.io</td><td class="alertOK">GREEN</td><td>Ping successful!</td></tr>
<tr><td>ESXi</td><td>sfo01-w01-esx03.sfo.rainpole.io</td><td class="alertOK">GREEN</td><td>API Connectivity check successful!</td></tr>
<tr><td>ESXi</td><td>sfo01-w01-esx03.sfo.rainpole.io</td><td>-</td><td>** SSH status is disabled.</td></tr>
<tr><td>ESXi</td><td>sfo01-w01-esx04.sfo.rainpole.io</td><td class="alertOK">GREEN</td><td>API Connectivity check successful!</td></tr>
<tr><td>ESXi</td><td>sfo01-w01-esx04.sfo.rainpole.io</td><td class="alertOK">GREEN</td><td>Ping successful!</td></tr>
<tr><td>ESXi</td><td>sfo01-w01-esx04.sfo.rainpole.io</td><td>-</td><td>** SSH status is disabled.</td></tr>
<tr><td>NSX</td><td>sfo-m01-nsx01.sfo.rainpole.io</td><td class="alertOK">GREEN</td><td>API Connection check successful!</td></tr>
<tr><td>NSX</td><td>sfo-m01-nsx01.sfo.rainpole.io</td><td class="alertOK">GREEN</td><td>Ping successful!</td></tr>
<tr><td>NSX</td><td>sfo-m01-nsx01a.sfo.rainpole.io</td><td class="alertOK">GREEN</td><td>API Connection check successful!</td></tr>
<tr><td>NSX</td><td>sfo-m01-nsx01b.sfo.rainpole.io</td><td class="alertOK">GREEN</td><td>API Connection check successful!</td></tr>
<tr><td>NSX</td><td>sfo-m01-nsx01c.sfo.rainpole.io</td><td class="alertOK">GREEN</td><td>API Connection check successful!</td></tr>
<tr><td>NSX</td><td>sfo-w01-nsx01.sfo.rainpole.io</td><td class="alertOK">GREEN</td><td>Ping successful!</td></tr>
<tr><td>NSX</td><td>sfo-w01-nsx01.sfo.rainpole.io</td><td class="alertOK">GREEN</td><td>API Connection check successful!</td></tr>
<tr><td>NSX</td><td>sfo-w01-nsx01a.sfo.rainpole.io</td><td class="alertOK">GREEN</td><td>API Connection check successful!</td></tr>
<tr><td>NSX</td><td>sfo-w01-nsx01b.sfo.rainpole.io</td><td class="alertOK">GREEN</td><td>API Connection check successful!</td></tr>
<tr><td>NSX</td><td>sfo-w01-nsx01c.sfo.rainpole.io</td><td class="alertOK">GREEN</td><td>API Connection check successful!</td></tr>
<tr><td>vCenter</td><td>sfo-m01-vc01.sfo.rainpole.io</td><td class="alertOK">GREEN</td><td>API Connection check successful!</td></tr>
<tr><td>vCenter</td><td>sfo-m01-vc01.sfo.rainpole.io</td><td>-</td><td>** SSH status is enabled.</td></tr>
<tr><td>vCenter</td><td>sfo-m01-vc01.sfo.rainpole.io</td><td class="alertOK">GREEN</td><td>Ping successful!</td></tr>
<tr><td>vCenter</td><td>sfo-w01-vc01.sfo.rainpole.io</td><td class="alertOK">GREEN</td><td>Ping successful!</td></tr>
<tr><td>vCenter</td><td>sfo-w01-vc01.sfo.rainpole.io</td><td>-</td><td>** SSH status is enabled.</td></tr>
<tr><td>vCenter</td><td>sfo-w01-vc01.sfo.rainpole.io</td><td class="alertOK">GREEN</td><td>API Connection check successful!</td></tr>
<tr><td>VRA</td><td>xint-vra01a.rainpole.io</td><td class="alertOK">GREEN</td><td>Ping successful!</td></tr>
<tr><td>VRA</td><td>xint-vra01a.rainpole.io</td><td>-</td><td>** SSH status is enabled.</td></tr>
<tr><td>VRA</td><td>xint-vra01b.rainpole.io</td><td>-</td><td>** SSH status is enabled.</td></tr>
<tr><td>VRA</td><td>xint-vra01b.rainpole.io</td><td class="alertOK">GREEN</td><td>Ping successful!</td></tr>
<tr><td>VRA</td><td>xint-vra01c.rainpole.io</td><td class="alertOK">GREEN</td><td>Ping successful!</td></tr>
<tr><td>VRA</td><td>xint-vra01c.rainpole.io</td><td>-</td><td>** SSH status is enabled.</td></tr>
<tr><td>vRLI</td><td>sfo-vrli01a.sfo.rainpole.io</td><td class="alertOK">GREEN</td><td>Ping successful!</td></tr>
<tr><td>VRLI</td><td>sfo-vrli01a.sfo.rainpole.io</td><td>-</td><td>** SSH status is enabled.</td></tr>
<tr><td>vRLI</td><td>sfo-vrli01b.sfo.rainpole.io</td><td class="alertOK">GREEN</td><td>Ping successful!</td></tr>
<tr><td>VRLI</td><td>sfo-vrli01b.sfo.rainpole.io</td><td>-</td><td>** SSH status is enabled.</td></tr>
<tr><td>vRLI</td><td>sfo-vrli01c.sfo.rainpole.io</td><td class="alertOK">GREEN</td><td>Ping successful!</td></tr>
<tr><td>VRLI</td><td>sfo-vrli01c.sfo.rainpole.io</td><td>-</td><td>** SSH status is enabled.</td></tr>
<tr><td>vROps</td><td>sfo-vropsc01a.sfo.rainpole.io</td><td class="alertOK">GREEN</td><td>Ping successful!</td></tr>
<tr><td>vROps</td><td>sfo-vropsc01a.sfo.rainpole.io</td><td>-</td><td>** SSH status is enabled.</td></tr>
<tr><td>vROps</td><td>sfo-vropsc01b.sfo.rainpole.io</td><td class="alertOK">GREEN</td><td>Ping successful!</td></tr>
<tr><td>vROps</td><td>sfo-vropsc01b.sfo.rainpole.io</td><td>-</td><td>** SSH status is enabled.</td></tr>
<tr><td>vROps</td><td>xint-vrops01a.rainpole.io</td><td>-</td><td>** SSH status is enabled.</td></tr>
<tr><td>vROps</td><td>xint-vrops01a.rainpole.io</td><td class="alertOK">GREEN</td><td>Ping successful!</td></tr>
<tr><td>vROps</td><td>xint-vrops01b.rainpole.io</td><td>-</td><td>** SSH status is enabled.</td></tr>
<tr><td>vROps</td><td>xint-vrops01b.rainpole.io</td><td class="alertOK">GREEN</td><td>Ping successful!</td></tr>
<tr><td>vROps</td><td>xint-vrops01c.rainpole.io</td><td class="alertOK">GREEN</td><td>Ping successful!</td></tr>
<tr><td>vROps</td><td>xint-vrops01c.rainpole.io</td><td>-</td><td>** SSH status is enabled.</td></tr>
<tr><td>VRSLCM</td><td>xint-vrslcm01.rainpole.io</td><td class="alertOK">GREEN</td><td>Ping successful!</td></tr>
<tr><td>VRSLCM</td><td>xint-vrslcm01.rainpole.io</td><td>-</td><td>** SSH status is enabled.</td></tr>
<tr><td>WSA</td><td>xint-wsa01a.rainpole.io</td><td>-</td><td>** SSH status is enabled.</td></tr>
<tr><td>WSA</td><td>xint-wsa01a.rainpole.io</td><td class="alertOK">GREEN</td><td>Ping successful!</td></tr>
<tr><td>WSA</td><td>xint-wsa01b.rainpole.io</td><td class="alertOK">GREEN</td><td>Ping successful!</td></tr>
<tr><td>WSA</td><td>xint-wsa01b.rainpole.io</td><td>-</td><td>** SSH status is enabled.</td></tr>
<tr><td>WSA</td><td>xint-wsa01c.rainpole.io</td><td class="alertOK">GREEN</td><td>Ping successful!</td></tr>
<tr><td>WSA</td><td>xint-wsa01c.rainpole.io</td><td>-</td><td>** SSH status is enabled.</td></tr>
```

image

**Breaking Changes?**

- [ ] Yes, there are breaking changes.
- [x] No, there are no breaking changes.

### CODE REVIEW

File: VMware.CloudFoundation.Reporting.psd1 [hunk 1]
```diff
@@ -12,7 +12,7 @@
     RootModule = '.\VMware.CloudFoundation.Reporting.psm1'
     
     # Version number of this module.
-    ModuleVersion = '2.1.0.1002'
+    ModuleVersion = '2.1.0.1003'
```
Comment (GaryJBlake) [inline VMware.CloudFoundation.Reporting.psd1:15]: Your merge details saying update 1003 > 1004 but here its 1002 > 1003 ?

File: VMware.CloudFoundation.Reporting.psm1 [hunk 2]
```diff
@@ -1145,6 +1145,91 @@ Function Publish-ConnectivityHealth {
 }
 Export-ModuleMember -Function Publish-ConnectivityHealth
 
+Function Publish-PingConnectivityHealth {
+    <#
+        .SYNOPSIS
+        Formats the PingConnectivity Health data from the SoS JSON output.
```
Comment (GaryJBlake) [inline VMware.CloudFoundation.Reporting.psm1:1151]:
```suggestion
        Formats the Ping Connectivity Health data from the SoS JSON output.
```

File: VMware.CloudFoundation.Reporting.psm1 [hunk 3]
```diff
@@ -1145,6 +1145,91 @@ Function Publish-ConnectivityHealth {
...
+
+        .DESCRIPTION
+        The Publish-PingConnectivityHealth cmdlet formats the Connectivity Health data from the SoS JSON output and
```
Comment (GaryJBlake) [inline VMware.CloudFoundation.Reporting.psm1:1154]:
```suggestion
        The Publish-PingConnectivityHealth cmdlet formats the Ping Connectivity Health data from the SoS JSON output and
```

**Review by GaryJBlake — CHANGES_REQUESTED** [review]

File: VMware.CloudFoundation.Reporting.psm1 (hunk 2 above)
Comment (tenthirtyam) [reply VMware.CloudFoundation.Reporting.psm1]: Resolved in 343e932d8ec8e2e0039d3a5ba19c568a5ef72754.

File: VMware.CloudFoundation.Reporting.psm1 (hunk 3 above)
Comment (tenthirtyam) [reply VMware.CloudFoundation.Reporting.psm1]: Resolved in 343e932d8ec8e2e0039d3a5ba19c568a5ef72754.

File: VMware.CloudFoundation.Reporting.psd1 (hunk 1 above)
Comment (tenthirtyam) [reply VMware.CloudFoundation.Reporting.psd1]: Resolved in 343e932d8ec8e2e0039d3a5ba19c568a5ef72754.

#131 needs to be reviewed and merged prior.

**Review by GaryJBlake — APPROVED** [review]
LGTM!

**Review by GaryJBlake — APPROVED** [review]
LGTM!

**Pull request merged**